action.yml

```diff
@@ -83,10 +83,14 @@ runs:
     - name: Run poetry update
       if: env.RUN_UPDATE != 'false'
       run: |
-        UPDATE_MESSAGE=$(poetry update | grep -i 'updating' | grep -v 'Updating dependencies' | sed 's/Updating//g')
-        echo $UPDATE_MESSAGE
-
-        echo "UPDATE_MESSAGE=$UPDATE_MESSAGE" >> $GITHUB_ENV
+        poetry install --no-interaction --no-root
+        poetry update | grep -i 'updating' | grep -v 'Updating dependencies' | sed 's/Updating//g' >> updated_deps.txt
```

Comment:
Should `cat updated_deps.txt` be in here twice?

Reply:
As in on 88 and 91? 

Yes, the first time is to output to the terminal, the second time the cat is captured by the write to $GITHUB_ENV and is used for setting the environment variable